src/Directory.Build.props

```diff
@@ -0,0 +1,41 @@
+<Project>
+
+  <PropertyGroup>
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+
+    <LangVersion>latest</LangVersion>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
probably add warninglevel 5 that was in the project before

Reply:
I did not see that other projects set this property so I decided to not keep it.

Reply:
There are one or two must haves that are in L5 (i.e. its not a noisy C++ style warning mode) so I tend to turn it on. But there's also enough noise to not make it a total no brainer.
Mentioning as this one had already reached L5